Cloud Application Manager/Troubleshooting/camdce-troubleshooting-tips.md

```diff
@@ -0,0 +1,289 @@
+
+**Solution**
+
+[Contact support](mailto:incident@CenturyLink.com). We will walk you through recovering the appliance. Send us the appliance ID, version number shown at the top of the [setup console](../Data Center Edition/camdce-initialsetup.md). And send us the logs you can download from the Logs section of the appliance setup console.
+
+Logs help us debug your appliance issues. The `.log` files include recent audit information like who did what, user connections, and contain the logs of all the services. If possible, copy the logs from `/data/logs` away from the CAM Appliance and have them ready for the CAM support team.
+
+### Resource sharing problems
+
+When trying to share a resource (Provider, Box, Instance) the desired user or team can't be found in the users list or the ownership of the resource cant be transfered to another user/team.
+
+**Cause 1**
+
+Resources can only be shared with other users or teams that belong to the same organization or have their organizations federated.
+
+**Solution**
+
+Make sure the resource owner and the desired user to share it with belong to the same organization or request the organization administrator to request the federation with the other organization though [Support](mailto:incident@CenturyLink.com).
+
+**Cause 2**
+
+Providers and Deployment policy boxes can only be shared with other users or teams that belong to the same cost center. Each user/team have an assigned costcenter that cannot be changed.
+
+**Solution 1**
+
+Make sure the resource owner and the desired user to share it with belong to the same costcenter. or request the organization administrator to request the federation with the other organization though [Support](mailto:incident@CenturyLink.com).
+
+**Solution 2**
+
+An organization administrator can edit the Costcenter and add any user in the organization (or federeated organizations) as a Costcenter administrator. Once the user is a costcenter administrator, any resource owned by any other user/team of the same costcenter could be shared with this user.
+
+**Cause 3**
+
+Providers and Deployment policy boxes ownership can only be transfered to other users or teams that belong to the same cost center. Each user/team have an assigned costcenter that cannot be changed.
+
+**Solution**
+
+Make sure the resource owner and the desired user to transfer ownership to belong to the same costcenter.
+
+### Instance fails to register in Cloud Application Manager
+
+**Cause**
+
+An instance registration (import) is waiting for the CAM agent to install, but the agent installation never seems to be completed.
+
+**Solution 1**
+
+Make sure the communication from the instance to be register to CAM appliance works correctly.
+
+1. Connect to the instance by SSH or RDP.
+
+2. Run a ping command to check basic connectivity:
```

Comment:
Telnet to cam.ctl.io 443 to check that the instance to be registered has connectivity over secure socket layer port.